lib/screen/clubs/clubs_screen.dart

```diff
@@ -29,41 +30,70 @@ class ClubsScreenState extends State<ClubsScreen> with BackNavigatorMixin, Error
           backgroundColor: theme.colorsTheme.backgroundDark,
           body: Center(
             child: LayoutBuilder(builder: (context, constraints) {
+              final background = Stack(
+                children: [
+                  Image.asset(
+                    'assets/images/icappsGroup.jpg',
+                    fit: BoxFit.cover,
+                    width: double.infinity,
+                    height: double.infinity,
+                  ),
+                  const AnimatedGradiant(gradiantRatio: 0.15),
+                ],
+              );
               if (constraints.maxWidth > 1100) {
-                return Row(
-                  mainAxisAlignment: MainAxisAlignment.center,
+                return Stack(
+                  alignment: Alignment.center,
                   children: [
-                    ClubCard(
-                      onClick: () => viewModel.onRunningCardClicked(viewModel.runningClub),
-                      title: localization.clubSelectionRunningTitle,
-                      url: viewModel.runningPicture,
-                    ),
-                    const SizedBox(width: ThemeDimens.padding128),
-                    ClubCard(
-                      onClick: () => viewModel.onCyclingCardClicked(viewModel.cyclingClub),
-                      title: localization.clubSelectionCyclingTitle,
-                      url: viewModel.cyclingPicture,
+                    background,
+                    Row(
+                      mainAxisAlignment: MainAxisAlignment.center,
+                      children: [
+                        ClubCard(
+                          onClick: () => viewModel.onRunningCardClicked(viewModel.runningClub),
```

Comment:
Your view model already knows which club you want to select. 
So change method without a parameter or make the running and cycling tapped same method with a parameter